src/app/core/signals/remote_signals/connector.nim

```diff
@@ -0,0 +1,17 @@
+import json, tables, chronicles
+import base
+
+include  app_service/common/json_utils
+
+type ConnectorSendRequestAccountsSignal* = ref object of Signal
+  url*: string
+  name*: string
+  iconUrl*: string
+  requestID*: string
+
+proc fromEvent*(T: type ConnectorSendRequestAccountsSignal, event: JsonNode): ConnectorSendRequestAccountsSignal =
+  result = ConnectorSendRequestAccountsSignal()
+  result.url = event["event"]{"url"}.getStr()
+  result.name = event["event"]{"name"}.getStr()
+  result.iconUrl = event["event"]{"iconUrl"}.getStr()
+  result.requestID = event["event"]{"requestId"}.getStr()
```

Comment:
In nim code i think it should be `requestId`, you can use my changes from here: https://github.com/status-im/status-desktop/pull/15640